### PR TITLE
Issue 264 - [use local references for internal libraries]

### LINKED ISSUE #264: use local references for internal libraries
Instead importing ODIM implemented libraries from github.conm, use local references(eg. ../lib-persistence...)

### PR DESCRIPTION
**Pull request by a-ajith**
The PR contains:

1. replace code snippet in go mod file for the internal libraries
2. removal of internal libraries from the go.sum files
3. removal of unused third party redisearch library from svc-task
4. Dockerfile changes for accommodating the local referencing.

closes #264

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

File: install/Docker/dockerfiles/scripts/build_odim.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+LIST=`ls -R | grep -v 'lib-rest-client' | grep -E '^svc-|^plugin-'`
+echo $LIST
+for i in $LIST; do
+    cd $i
+    go mod download
+    go mod vendor
+    go build -i -race .
+    if [ $? -eq 0 ]; then
+        echo Build for odimra service/lib dependencies $i are Successful !!!!
```
Comment (rahulroshan-kachchap) [inline install/Docker/dockerfiles/scripts/build_odim.sh:24]: Can we restructure the sentence? 
Something like "Successfully  build $i service"

File: install/Docker/dockerfiles/scripts/build_odim.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        arr1+=$i;
```
Comment (rahulroshan-kachchap) [inline install/Docker/dockerfiles/scripts/build_odim.sh:25]: are we using this array anywhere ?

File: install/Docker/dockerfiles/scripts/build_odim.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    else
+        echo Build for odimra service/lib dependency $i Failed !!!!
+        arr2+=$i;
```
Comment (rahulroshan-kachchap) [inline install/Docker/dockerfiles/scripts/build_odim.sh:28]: are we using this array

File: install/Docker/dockerfiles/scripts/build_odim.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        flag=0
```
Comment (rahulroshan-kachchap) [inline install/Docker/dockerfiles/scripts/build_odim.sh:29]: flag=0 is never used anywhere

**Review by rahulroshan-kachchap — CHANGES_REQUESTED** [review]

File: install/Docker/dockerfiles/scripts/build_odim.sh (hunk 1 above)
Comment (a-ajith) [reply install/Docker/dockerfiles/scripts/build_odim.sh]: Addressed.

File: install/Docker/dockerfiles/scripts/build_odim.sh (hunk 2 above)
Comment (a-ajith) [reply install/Docker/dockerfiles/scripts/build_odim.sh]: Addressed.

File: install/Docker/dockerfiles/scripts/build_odim.sh (hunk 3 above)
Comment (a-ajith) [reply install/Docker/dockerfiles/scripts/build_odim.sh]: Addressed.

File: install/Docker/dockerfiles/scripts/build_odim.sh (hunk 4 above)
Comment (a-ajith) [reply install/Docker/dockerfiles/scripts/build_odim.sh]: Addressed.

**Review by rahulroshan-kachchap — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Pull request merged**